modules/kargoBidAdapter.js

```diff
@@ -366,6 +371,19 @@ function getUserIds(tdidAdapter, usp, gdpr, eids) {
     userIds.sharedIDEids = eids;
   }
 
+  if (gpp) {
+    const parsedGPP = {}
```

Comment:
should this be `const`? it's getting modified below

Reply:
same as above.